src/server/game/DungeonFinding/LFGMgr.h

```diff
@@ -150,8 +152,8 @@ namespace lfg
 
     struct RBEntryInfo
     {
-        RBEntryInfo() {}
-        RBEntryInfo(uint8 _roles, std::string const& _comment) : roles(_roles), comment(_comment) {}
+        RBEntryInfo() = default;
+        RBEntryInfo(uint8 _roles, std::string  _comment) : roles(_roles), comment(std::move(_comment)) {}
```

Comment:
I just want to understand, why do you use std::move() in these places? What is the general purpose?

I was not going to ask but as I see it in several places I'd like to learn